tests/test_hash_table.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+
+from src.hash_table import ChainHashTable
+
+
+class TestChainHashTable(unittest.TestCase):
```

Comment:
We should also check for edge cases. For example, search for key which doesn't exist or remove from empty hash table

Reply:
For Python built-in hash table `dict`, it will return KeyError when trying to remove from empty table. 
```
d_ = dict()
d_.pop("8") # KeyError
```
Should we mimic this behavior or just return None like what I have done?

Reply:
I think it's up to you and @mindxhub/py team. I'm ok with current behavior.